src/index.js

```diff
@@ -286,6 +291,25 @@ function applyExpanded (css, opts) {
     css.raws.after = '\n';
 }
 
+function applyTransformFeatures (rule, opts) {
+    // hexadecimal color transformations
```

Comment:
Can you exit early instead of testing the rule type and color twice?

```js
if (rule.type !== 'decl') {
  return;
}
const isColor = isHexColor(rule.value);
if (!isColor) {
  return;
}
```